pkg/vulnsrc/vulnerability/vulnerability.go

```diff
@@ -179,6 +180,36 @@ func getReferences(details map[types.SourceID]types.VulnerabilityDetail) []strin
 	return refs
 }
 
+func getPublishedDate(vulnID string, details map[types.SourceID]types.VulnerabilityDetail) *time.Time {
+	// We should take PublishedData from
+	//  - NVD - for CVE-IDs
+	//  - GHSA - for GHSA-IDs
+	// cf. https://github.com/aquasecurity/trivy-db/issues/522
+	switch {
+	case strings.HasPrefix(vulnID, "CVE"):
```

Comment:
nit: The [crisis](https://therecord.media/cisa-extends-cve-program-contract-with-mitre) surrounding the CVE program has been a topic of discussion, and proposals such as [GCVE](https://gcve.eu/2025/04/16/gcve-allocation-announced/) have been made. Since new programs like CVEZ may also be proposed in the future, I think it's slightly safer to check all the way up to the hyphen.


```suggestion
	case strings.HasPrefix(vulnID, "CVE-"):
```

Reply:
It make sense. 
Updated in https://github.com/aquasecurity/trivy-db/pull/527/commits/7591542f48e29de1d6de81fdbc47023ab7f0ae9f